src/libexpr/value.hh

```diff
@@ -64,6 +64,8 @@ class JSONPlaceholder;
 
 typedef int64_t NixInt;
 typedef double NixFloat;
+typedef std::pair<Path, std::string> NixStringContextElem;
```

Comment:
Maybe remove this and use `NixStringContext::value_type` instead? Doesn’t change much, but I find it more explicit

Reply:
Or maybe it should be this way to be consistent with `SearchPathElem`. which I had completely forgotten about in the time since I first wrote this code.